### PR TITLE
small updates to 'ways to contribute' page

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

* remove link to dev mailing list which isn't used
* a few other clarifications

### Reviewer guidance

look good?

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by jonboiser — APPROVED** [review]

File: docs/contributing/ways_to_contribute.rst [hunk 1]
```diff
@@ -12,44 +12,47 @@ Talk to us
...
+If you have found a bug and are comfortable using Github and Markdown, you can create a `Github issue <https://github.com/learningequality/kolibri/issues>`__. Please search the existing issues first to see if it's already been reported, and make sure to use the template.
 
-* Your operating system name and version.
-* Any details about your local setup that might be helpful in troubleshooting.
-* Detailed steps to reproduce the bug.
+If you are reporting an issue, please include:
+
+* Your operating system name and version
+* Any details about your local setup that might be helpful in troubleshooting
+* Detailed steps to reproduce the bug
```
Comment (radinamatic) [inline docs/contributing/ways_to_contribute.rst]: Or simpler:

> If you are reporting an issue, please make sure to fill in all the relevant parts in the **New issue** [template](https://github.com/learningequality/kolibri/issues/new)...? 🙂

Comment (indirectlylit) [reply docs/contributing/ways_to_contribute.rst]: I do mention that above so you're right, this is redundant

Comment (indirectlylit) [reply docs/contributing/ways_to_contribute.rst]: updated

**Review by radinamatic — APPROVED** [review]
LGTM! 👍

**Pull request merged**